src/_micro-api/libraries/signalDecoder/src/signalDecoder.h

```diff
@@ -134,6 +134,8 @@ constexpr const uint8_t MSG_END = 3;
 
 enum status { searching, clockfound, syncfound, detecting, mcdecoding };
 
+char* myitoa(int num, char* str);   // selfmade myitoa function
```

Comment:
Gehört die nicht eher wo anders hin sowas wie functions.h?

Reply:
>Gehört die nicht eher wo anders hin sowas wie functions.h?

EIGENTOR!
Die Funktion ist nur im signalDecoder in Benutzung und sobald ich diese hier verschiebe, so geht der SignalDecoder nicht zu kompilieren weil er keine Abhänigigkeiten besitzt :-D ... (okay, er besitzt genau 3 aber nicht die von functions.h)

Reply:
ich hätte das nachsehen sollen, sehe ich ein.
1:0 für dich :)